### PR TITLE
Fix falsy data mapped to nil

### PR DESCRIPTION
**Pull request by qhuyduong**
Signed-off-by: Huy Duong <huy.duong@employmenthero.com>

Currently when there is a falsy data in the message (e.g: `active: false`), it will be mapped to nil instead.

This PR is to fix that.

### CODE REVIEW

File: lib/kafka_handle_event/attribute_mapper.rb [hunk 1]
```diff
@@ -11,40 +11,37 @@ def attributes
...
-        message[:data][external_column] || default_value
+      if message[:data] && !message[:data][external_column].nil?
+        message[:data][external_column]
       else
         default_value
       end
     end
 
     def get_block_map_value(mapper, message)
-      internal_column = mapper[0]
```
Comment (anlnt) [inline lib/kafka_handle_event/attribute_mapper.rb:44]: why do we remove this?

File: .gitignore [hunk 2]
```diff
@@ -48,3 +48,4 @@ build-iPhoneSimulator/
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+Gemfile.lock
```
Comment (anlnt) [inline .gitignore:51]: Why don't we need Gemfile.lock in this repo? 😃

File: lib/kafka_handle_event/attribute_mapper.rb (hunk 1 above)
Comment (qhuyduong) [reply lib/kafka_handle_event/attribute_mapper.rb:44]: It's not used any where in this method :smile:

File: .gitignore (hunk 2 above)
Comment (qhuyduong) [reply .gitignore:51]: This is best practice when developing a gem. Let the app using this gem resolve the dependency.
https://stackoverflow.com/a/4151540

Comment (anlnt) [reply .gitignore:51]: Good to know. I don't have much experience developing a gem 😃

File: lib/kafka_handle_event/attribute_mapper.rb [hunk 3]
```diff
@@ -11,40 +11,37 @@ def attributes
...
+                                  else
+                                    get_attrbute_map_value(mapper, message)
+                                  end
       end
       attributes
     end
 
     private
 
     def get_attrbute_map_value(mapper, message)
```
Comment (khacluan) [inline lib/kafka_handle_event/attribute_mapper.rb:32]: typo `attrbute`

**Comment by qhuyduong** [comment]
@khacluan @anlnt please help to review again

**Pull request merged**